src/compiler/compiler.rs

```diff
@@ -1148,7 +1148,7 @@ mod test {
     use crate::mock_command::*;
     use crate::test::mock_storage::MockStorage;
     use crate::test::utils::*;
-    use std::fs::{self, File};
+    use crate::util::fs::{self, File};
```

Comment:
I'd be in favour of never using `File` but always stick to `fs::File` everywhere. There are almost no files with more than 2 occurences of `File`.

Reply:
This is more of a code style issue (arguments go both ways) and I'd rather avoid the churn of making changes for the sake of it. Let's keep the diff to a minimum and focus on the easily swapping out `std::fs` for better errors, which is the reason for this PR.

Reply:
Fair point.